grammars/css.cson

```diff
@@ -494,17 +494,15 @@
         ]
       }
       {
-        'begin': '(:)\\s*'
+        'begin': ':'
```

Comment:
This should be kept as `(:)\\s*` so that extra whitespace isn't scoped.  Same goes for the end pattern.